aries-site/src/pages/templates/user-feedback-collection.mdx

```diff
@@ -0,0 +1,142 @@
+import { FeedbackAnatomy, SimpleFeedbackExample, ModalTaskFlowExample } from '../../examples';
+import { Example } from '../../layouts';
+import { Checkmark } from 'grommet-icons';
+
+export const contact = {
+  name: 'Doris Singer',
+  email: 'doris.singer@hpe.com',
+};
+
+Feedback collected from users provides qualitative insights into users' perception of ease-of-use 
+and value delivered. Qualitative measures compliment quantitative usage and interaction data 
+to form a well-rounded analytic picture of the customer experience.
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/feedback/SimpleFeedbackExample.js"
+  docs="https://v2.grommet.io/starrating?theme=hpe"
+  figma="https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5554%3A77765&t=TNCIYkwaIfFrdWxf-1"
+>
+  <SimpleFeedbackExample />
+</Example>
+
+## When to use
+When and where to collect feedback from users should be a collaborative effort working with the <a href={`mailto:${contact.email}?subject=Collecting%20user%20feedback%20in%20a%20product%20experience`}>unified customer experience team</a>.
+This team will work with you to construct a questionnaire aligned with your objectives and 
+[provide implementation details](#feedback-setup-and-submission) such as API credentials and endpoints for submitting 
+feedback responses.
+
+In-app feedback collection should be considered where it aids the understanding of users' 
+perceptions around the ease-of-use and/or satisfaction with particular content, features, 
+or workflows.
+
+
+Feedback may be [actively solicited](#solicited-feedback) or [passively collected](#capturing-unsolicited-feedback).
+
+### When not to use
+Don’t collect feedback in the following use cases:
+- If it disrupts a user flow or is intrusive in any way.
+- There is no actionable plan for what to do with the data.
+- In locations where users do not have enough context to form a valid opinion. 
```

Comment:
Is it possible for us to elaborate a little bit more on this. Is there a couple of examples we can provide to let our users know where Feedback shouldn't be utilized? Like in the middle of a check out form? On a detailed view page? Just thinking out loud because I found myself asking when would be a few scenarios where I wouldn't be able to use Feedback.

Reply:
Maybe we can include some Do's and Don't examples within this section? This can be something we do later on.

Reply:
sounds good I can add that to the follow on ticket we have for next steps